.github/workflows/actions.yaml

```diff
@@ -24,7 +41,7 @@ jobs:
       - name: Run Go Security
         uses: securego/gosec@master
```

Comment:
Did using an older version of gosec not work?

Reply:
nope

Reply:
hmm something is off, I see that you changed it in file, but in gosec report, it says version was:
```
Summary:
  Gosec  : v2.9.4
  Files  : 14
  Lines  : 5965
  Nosec  : 1
  Issues : 7
```

Reply:
looking at past runs, we can see this error didn't happen when we used gosec 2.9.3
```
Summary:
  Gosec  : v2.9.3
  Files  : 14
  Lines  : 5965
  Nosec  : 8
  Issues : 0
```

in 2.9.3, it's able to read all 8 NoSecs, but in newest version, 2.9.4, it only reads 1 and misses 7

Reply:
we can see that starting yesterday, https://github.com/dell/csi-powerflex/pull/58/commits/3761d47570669b9c0c21538edfc36aef2eadda4d
was when the gosec version changed and we got the error